sdks/python/apache_beam/testing/test_stream.py

```diff
@@ -171,13 +180,14 @@ class TestStream(PTransform):
   time. After all of the specified elements are emitted, ceases to produce
   output.
   """
```

Comment:
Why do we need to declare output_tags here?
Are you trying to allow for outputs that have no events, otherwise shouldn't the tags come from the list of events?

The answer here impacts what we should be doing in expand in the no output_tags case in expand below.

Reply:
> Are you trying to allow for outputs that have no events, otherwise shouldn't the tags come from the list of events?

Yep! The TestStreamService will allow users to define a TestStream with the output_tags specified at creation time and the events supplied at runtime.